core/CMakeLists.txt

```diff
@@ -117,10 +117,17 @@ target_compile_definitions(Core PRIVATE
 if (runtime_cxxmodules)
   list(APPEND core_implicit_modules "-mSystemByproducts")
   # Force generation of _Builtin_intrinsics from Core.
-  list(APPEND core_implicit_modules "-m" "_Builtin_intrinsics" "-mByproduct" "_Builtin_intrinsics")
-  list(APPEND core_implicit_modules "-mByproduct" "ROOT_Foundation_Stage1_NoRTTI")
-  list(APPEND core_implicit_modules "-mByproduct" "ROOT_Foundation_C")
-  list(APPEND core_implicit_modules "-mByproduct" "ROOT_Rtypes")
+  list(APPEND core_implicit_modules "-m" "_Builtin_intrinsics")
```

Comment:
This looks like it would now be listed twice in `core_implicit_modules`, i.e. once more inside the new foreach loop.